src/vimageHosting.cpp

```diff
@@ -0,0 +1,680 @@
+#include "vimageHosting.h"
```

Comment:
Please rename it to `vimagehosting.h` and `vimagehosting.cpp`. Source files should be all in lower case name.